woostarter/patterns/about-left-aligned-image-right-text-dark.php

```diff
@@ -17,7 +17,7 @@
 <!-- wp:columns {"verticalAlignment":"center","align":"wide","style":{"spacing":{"blockGap":{"left":"var:preset|spacing|50"}}}} -->
 <div class="wp-block-columns alignwide are-vertically-aligned-center"><!-- wp:column {"verticalAlignment":"center"} -->
 <div class="wp-block-column is-vertically-aligned-center"><!-- wp:image {"id":61,"aspectRatio":"1","scale":"cover","sizeSlug":"full","linkDestination":"none"} -->
-<figure class="wp-block-image size-full"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/wp-content/themes/woostarter/assets/images/coming-soon-img.webp" alt="A person holding a rustic, handmade ceramic mug in a cozy, dimly lit setting." class="wp-image-61" style="aspect-ratio:1;object-fit:cover"/></figure>
+<figure class="wp-block-image size-full"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/coming-soon-img.webp" alt="A person holding a rustic, handmade ceramic mug in a cozy, dimly lit setting." class="wp-image-61" style="aspect-ratio:1;object-fit:cover"/></figure>
```

Comment:
Should the alt text be translatable here? I've noticed there a few other cases where similar alt text is not wrapped in a translation function too.

Are we going to address things like this as part of https://github.com/woocommerce/woo-themes/issues/67 later on?